source/Junkership.js

```diff
@@ -87,11 +98,21 @@ export default class Junkership extends Pixi.Sprite {
         this.hitBox.y = this.y + 1
     }
 
-    shoot() {
-        game.addChild(new Projectile(
-            this.x + this.width,
-            this.y + this.height/2,
-            this))
+    fire(curXpos,curYpos,curShip,wOffset,hOffset) {
+        if(this.powerUp.curPower == "peaShoota") {
```

Comment:
Given the amount of weapons we're going to have, this is going to get ugly real fast. Do you have a plan in mind for streamlining this in the future? I'd rather not run a couple of dozen if statements if we can help it.